app/src/androidTest/java/com/MainActivityTest.java

```diff
@@ -43,7 +43,13 @@ public void performTest() throws FaceProcessingException {
         float[][] result = new float[numOfPhotos][];
         Mat[] photos = new Mat[numOfPhotos];
         NeuralModel model = GlobalData.getModel(appContext,
-                appContext.getResources().getString(R.string.model_Facenet)).neuralModel;
+                GlobalData.getUserSettings(appContext).getString(
+                        appContext.getResources().getString(R.string.user_Settings_user_model_key),
+                        appContext.getResources().getString(R.string.default_model_name)),
+                GlobalData.getUserSettings(appContext).getString(
+                        appContext.getResources().getString(R.string.user_Settings_user_model_key),
```

Comment:
The second string you pass here is the same as first (R.string.user_Settings_user_model_key)
Also chaining it this way is very illegible.